<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-04T15:03:35Z" name="TestingDemo1" finished-at="2021-03-04T15:03:35Z" duration-ms="54">
    <groups>
    </groups>
    <test started-at="2021-03-04T15:03:35Z" name="TestNG tests" finished-at="2021-03-04T15:03:35Z" duration-ms="54">
      <class name="grades.GradeSchemeNGTest">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="setUpClass" finished-at="2021-03-04T09:03:35Z" duration-ms="20" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method is-config="true" signature="setUpMethod()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="setUpMethod" finished-at="2021-03-04T09:03:35Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="testGetGrade()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="testGetGrade" finished-at="2021-03-04T09:03:35Z" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetGrade -->
        <test-method is-config="true" signature="tearDownMethod()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="tearDownMethod" finished-at="2021-03-04T09:03:35Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method signature="testIsFreezing()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="testIsFreezing" finished-at="2021-03-04T09:03:35Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at org.testng.Assert.assertEquals(Assert.java:296)
	at grades.GradeSchemeNGTest.testIsFreezing(GradeSchemeNGTest.java:67)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIsFreezing -->
        <test-method is-config="true" signature="setUpMethod()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="setUpMethod" finished-at="2021-03-04T09:03:35Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method is-config="true" signature="tearDownMethod()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="tearDownMethod" finished-at="2021-03-04T09:03:35Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:grades.GradeSchemeNGTest@21a947fe]" started-at="2021-03-04T09:03:35Z" name="tearDownClass" finished-at="2021-03-04T09:03:35Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- grades.GradeSchemeNGTest -->
    </test> <!-- TestNG tests -->
  </suite> <!-- TestingDemo1 -->
</testng-results>
